consensus/polybft/state.go

```diff
@@ -183,48 +51,25 @@ type TransportMessage struct {
 	EpochNumber uint64
 }
 
-func (t *TransportMessage) ToSignature() *MessageSignature {
-	return &MessageSignature{
-		Signature: t.Signature,
-		From:      t.NodeID,
-	}
-}
-
 var (
-	// bucket to store rootchain bridge events
-	syncStateEventsBucket = []byte("stateSyncEvents")
-	// bucket to store exit contract events
-	exitEventsBucket = []byte("exitEvent")
-	// bucket to store commitments
-	commitmentsBucket = []byte("commitments")
-	// bucket to store state sync proofs
-	stateSyncProofsBucket = []byte("stateSyncProofs")
-	// bucket to store epochs and all its nested buckets (message votes and message pool events)
-	epochsBucket = []byte("epochs")
-	// bucket to store message votes (signatures)
-	messageVotesBucket = []byte("votes")
-	// bucket to store validator snapshots
-	validatorSnapshotsBucket = []byte("validatorSnapshots")
-	// bucket to store proposer calculator snapshot
-	proposerCalcSnapshotBucket = []byte("proposerCalculatorSnapshot")
 	// array of all parent buckets
 	parentBuckets = [][]byte{syncStateEventsBucket, exitEventsBucket, commitmentsBucket, stateSyncProofsBucket,
```

Comment:
We could move bucket creation to each store, and let it create its own bucket. It would mean, every store would create a db transaction on node startup, but we would have a more separated code. What do you think?